c/g6-read-iterator.c

```diff
@@ -124,9 +123,10 @@ int beginG6ReadIterationFromG6FilePath(G6ReadIterator *pG6ReadIterator, char *g6
 
 	if (g6Infile == NULL)
 	{
-		char * formatStr = "Unable to open .g6 file with path \"%.*s\"\n";
-		sprintf(Line, formatStr, (int) (MAXLINE - strlen(formatStr)), g6FilePath);
-		ErrorMessage(Line);
+		char * messageFormat = "Unable to open .g6 file with path \"%.*s\"\n";
+		char messageContents[MAXLINE + 1];
+		sprintf(messageContents, messageFormat, (int) (MAXLINE - strlen(messageFormat)), g6FilePath);
```

Comment:
For consistency, may as well do int charsAvailForFilename declaration, as you did in  code below.